dts/bindings/modem/gsm-ppp.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 G-Technologies Sdn. Bhd.
+# SPDX-License-Identifier: Apache-2.0
+
+description: GSM PPP modem
+
+compatible: "gsm,ppp"
```

Comment:
I wonder if `gsm,ppp` is a good compatible string. Maybe it should be `zephyr,gsm-ppp` instead?

@mbolivar-nordic @galak ?

Reply:
#35422 was just merged. I guess this means that we should use `zephyr,` prefix.